src/Matrix.cc

```diff
@@ -2544,6 +2545,61 @@ NAN_METHOD(Matrix::Mean) {
   info.GetReturnValue().Set(arr);
 }
 
+// http://docs.opencv.org/2.4/modules/imgproc/doc/filtering.html#copymakeborder
+NAN_METHOD(Matrix::CopyMakeBorder) {
+  SETUP_FUNCTION(Matrix)
+
+  double t = info[0]->NumberValue();
+  double b = info[1]->NumberValue();
+  double l = info[2]->NumberValue();
+  double r = info[3]->NumberValue();
+
+  int borderType = cv::BORDER_DEFAULT;
+  cv::Scalar value;
+  if (info.Length() > 4) {
+    borderType = info[4]->IntegerValue();
+    value = cv::Scalar(0, 0, 0, 0);
+    if (borderType == cv::BORDER_CONSTANT) {
+      if (!info[5]->IsArray()) {
```

Comment:
Is there a way to allow for a `cv::Scalar` object to be passed instead? This would eliminate almost all of this code.